oracle/CMakeLists.txt

```diff
@@ -23,7 +23,9 @@ SET(oracle_SOURCES
     ../cockatrice/src/settings/layoutssettings.cpp
     ../cockatrice/src/thememanager.cpp 
     ../cockatrice/src/qt-json/json.cpp
- )
+    ../cockatrice/src/releasechannel.cpp
+     ${VERSION_STRING_CPP}
```

Comment:
version cpp wasn't here already? How silly. We should probably make cockatrice's about menu reusable and call the same code from cockatrice and oracle. Not in this pr though